CONTRIBUTING.md

```diff
@@ -0,0 +1,60 @@
+# How to Contribute
+
+Contributing is easy: just change the doc and create a pull request.
```

Comment:
We may need to link to a basic guide describing the need to create a new branch first. I messed up when I first contributed here. At that time I made my changes directly on my master branch, submitted a PR, merged back changes from "upstream" master (which was via a merge commit) and then proceeded to make more changes and then finally tried to create a PR from the resulting mess.

I ended up just nuking my fork and recreating it. After doing some reading I realized where I went wrong and I think I've been OK since. Even if we don't have a solid guide now to link to, maybe make a reference to the required process?

```diff
-Contributing is easy: just change the doc and create a pull request.
+Contributing is easy: fork this repo, create a new branch, change the doc and then create a pull request.
```

It can probably be said simpler and I imagine it will be once we have a small doc to point users to. At that point we can say something like:

```diff
-Contributing is easy: just change the doc and create a pull request.
+Contributing is easy: follow the brief steps on the XYZ page to make changes and submit them for review
```

Reply:
This doc here comes up when you do a PR on github. I didn't think about this before, but maybe we should in the future limit it to what you need to know at that instant, with the major of the contribution doc being a separate page. Nevertheless, that's for another shot, let's get the info first in at all.